store/store_test.go

```diff
@@ -90,6 +90,30 @@ func TestEDSStore(t *testing.T) {
 		ensureAmountFileAndLinks(t, dir, 2, 1)
 	})
 
+	t.Run("Second Put after partial write", func(t *testing.T) {
+		dir := t.TempDir()
```

Comment:
In case tests are run in parallel, do they have their own temp dir?

Reply:
yes

Reply:
Also test coverage is going to be greatly improved in following PR